ts/components/Pinpad/index.tsx

```diff
@@ -72,11 +75,33 @@ class Pinpad extends React.PureComponent<Props, State> {
     ]
   ];
 
+  private shake() {
```

Comment:
prefer lamba style methods

```suggestion
  private shake = () => {
```